public/images/account/org-benefits/2.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32">
+  <g id="control-3" transform="translate(-1 -1)">
+    <path id="路径_15066" data-name="路径 15066" d="M32,17.185v-.371a1.86,1.86,0,0,0-1.86-1.86h-.278A1.887,1.887,0,0,1,28.07,13.59a11.5,11.5,0,0,0-.829-2.006,1.891,1.891,0,0,1,.3-2.235l.194-.194a1.86,1.86,0,0,0,0-2.63l-.262-.262a1.86,1.86,0,0,0-2.63,0l-.194.194a1.891,1.891,0,0,1-2.235.3A11.5,11.5,0,0,0,20.41,5.93a1.887,1.887,0,0,1-1.365-1.792V3.86A1.86,1.86,0,0,0,17.185,2h-.371a1.86,1.86,0,0,0-1.86,1.86v.278A1.887,1.887,0,0,1,13.59,5.93a11.5,11.5,0,0,0-2.006.829,1.891,1.891,0,0,1-2.235-.3l-.194-.194a1.86,1.86,0,0,0-2.63,0l-.262.262a1.86,1.86,0,0,0,0,2.63l.194.194a1.891,1.891,0,0,1,.3,2.235A11.5,11.5,0,0,0,5.93,13.59a1.887,1.887,0,0,1-1.792,1.365H3.86A1.86,1.86,0,0,0,2,16.815v.371a1.86,1.86,0,0,0,1.86,1.86h.278A1.887,1.887,0,0,1,5.93,20.41a11.5,11.5,0,0,0,.829,2.006,1.891,1.891,0,0,1-.3,2.235l-.194.194a1.86,1.86,0,0,0,0,2.63l.262.262a1.86,1.86,0,0,0,2.63,0l.194-.194a1.891,1.891,0,0,1,2.235-.3,11.5,11.5,0,0,0,2.006.829,1.887,1.887,0,0,1,1.365,1.792v.278A1.86,1.86,0,0,0,16.815,32h.371a1.86,1.86,0,0,0,1.86-1.86v-.278A1.887,1.887,0,0,1,20.41,28.07a11.5,11.5,0,0,0,2.006-.829,1.891,1.891,0,0,1,2.235.3l.194.194a1.86,1.86,0,0,0,2.63,0l.262-.262a1.86,1.86,0,0,0,0-2.63l-.194-.194a1.891,1.891,0,0,1-.3-2.235,11.5,11.5,0,0,0,.829-2.006,1.887,1.887,0,0,1,1.792-1.365h.278A1.86,1.86,0,0,0,32,17.185Z" fill="none" stroke="#fff" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" stroke-width="2"/>
+    <circle id="椭圆_1288" data-name="椭圆 1288" cx="5.455" cy="5.455" r="5.455" transform="translate(11.545 11.545)" fill="none" stroke="#fff" stroke-linecap="round" stroke-linejoin="round" stroke-miterlimit="10" stroke-width="2"/>
+  </g>
+</svg>
```

Comment:
The SVG loader (`@svgr/webpack`) may optimize the SVG size in build time or in runtime, but I suggest we can manually compress the SVG files before submitting them to the codebase. Just for saving some repo size. I usually to that job by https://vecta.io/nano